pipeline/outputs/cloudwatch.md

```diff
@@ -44,6 +46,55 @@ In your main configuration file append the following _Output_ section:
     auto_create_group On
 ```
 
+<a name="MetricsTutorial">Metrics Tutorial</a>
```

Comment:
So the problem with this is that when you view it, the phrase "Metric Tutorial" is the same size as the overall text. It does not look like a heading. So either add a header tag, or just do `### Metrics Tutorial` and forget about trying to make it a link.

Reply:
Updated as general heading.